test/src/helperTest.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+const expect = require("chai").expect;
+const os = require("os");
+const Helper = require("../../src/helper");
+
+describe("Helper", function() {
+	describe("#expandHome", function() {
+		it("should correctly expand a Unix path", function() {
+			expect([`${os.homedir()}/tmp`, `${os.homedir()}\\tmp`])
```

Comment:
This test seems completely useless, did you miss something?

Reply:
Yes I did :)
Good catch, will fix, thanks!

Reply:
Fixed.